cli/info/info.go

```diff
@@ -44,9 +47,35 @@ func infoMain(args []string, c cli.Config) (err error) {
 	if err != nil {
 		return err
 	}
+
+	_, err = helpers.ParseCertificatePEM(certPem)
+	if err != nil {
+		return errors.Wrap(errors.CertificateError, errors.DecodeFailed, err)
```

Comment:
Thanks for testing this. I think the err is already a cfssl error, so double wrapping it would trigger an panic here.